README.md

```diff
@@ -1,87 +1,94 @@
-	<a href="https://github.com/riivx/vendetta/contributors"><img src="https://img.shields.io/github/contributors/riivx/vendetta?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
+ <a href="https://github.com/catppuccin/vendetta/stargazers"><img src="https://img.shields.io/github/stars/catppuccin/vendetta?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
+ <a href="https://github.com/catppuccin/vendetta/issues"><img src="https://img.shields.io/github/issues/catppuccin/vendetta?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
+ <a href="https://github.com/catppuccin/vendetta/contributors"><img src="https://img.shields.io/github/contributors/catppuccin/vendetta?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
 </p>
 
 <p align="center">
-	<img src="https://raw.githubusercontent.com/riivx/vendetta/main/assets/preview.webp"/>
+  <img src="assets/preview.webp" height="700px"/>
 </p>
 
 ## Previews
 
 <details>
 <summary>🌻 Latte</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta/main/assets/latte.webp"/>
+<img src="assets/latte.webp"/>
 </details>
 <details>
 <summary>🪴 Frappé</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta/main/assets/frappe.webp"/>
+<img src="assets/frappe.webp"/>
 </details>
 <details>
 <summary>🌺 Macchiato</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta/main/assets/macchiato.webp"/>
+<img src="assets/macchiato.webp"/>
 </details>
 <details>
 <summary>🌿 Mocha</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta//main/assets/mocha.webp"/>
+<img src="assets/mocha.webp"/>
 </details>
 
 ## Usage
 
->[!WARNING]
->Vendetta was **discontinued** on February 7. 2024. Use [Bunny](https://github.com/pyoncord/Bunny) instead. It supports Vendetta themes
+> [!IMPORTANT]  
+> This is the home for the shared theme format that [†
+> Vendetta](https://github.com/vendetta-mod/Vendetta),
+> [Enmity](https://github.com/enmity-mod/enmity) &
+> [Bunny](https://github.com/pyoncord/Bunny) use.
+> Vendetta is no longer included in the instructions below, as it has reached
+> "End of Life."
 
 1. Choose a flavor, and copy its link below:
-	- **Latte**: https://raw.githubusercontent.com/catppuccin/vendetta/main/latte.json
-	- **Frappe**: https://raw.githubusercontent.com/catppuccin/vendetta/main/frappe.json
-	- **Macchiato**: https://raw.githubusercontent.com/catppuccin/vendetta/main/macchiato.json
-	- **Mocha**:  https://raw.githubusercontent.com/catppuccin/vendetta/main/mocha.json
-2. Head to `You Tab (aka Settings)` > `Vendetta` > `Themes`
+   - **Latte**: <https://raw.githubusercontent.com/catppuccin/vendetta/main/latte.json>
+   - **Frappe**: <https://raw.githubusercontent.com/catppuccin/vendetta/main/frappe.json>
+   - **Macchiato**: <https://raw.githubusercontent.com/catppuccin/vendetta/main/macchiato.json>
+   - **Mocha**: <https://raw.githubusercontent.com/catppuccin/vendetta/main/mocha.json>
```

Comment:
I think this step should be changed to the following:

```md
1. Choose a flavor + accent combination from the [themes](./themes) directory and copy the URL in the following format: `https://raw.githubusercontent.com/catppuccin/vendetta/main/themes/<flavor>/<accent>.json`
	
	For example, if you'd like to use the `mocha` flavor with the `mauve` accent, copy the following link:

	```
	<https://raw.githubusercontent.com/catppuccin/vendetta/main/themes/mocha/mauve.json>
	```
```

Doesn't have to be exactly that ^ but something similar and equivalent to where we showcase the format and give one example.

Reply:
> Doesn't have to be exactly that ^ but something similar and equivalent to where we showcase the format and give one example.

imo it should be exactly that